huggingface/pytorch/buildspec-neuron.yml

```diff
@@ -34,9 +34,9 @@ images:
     device_type: &DEVICE_TYPE neuron
     python_version: &DOCKER_PYTHON_VERSION py3
     tag_python_version: &TAG_PYTHON_VERSION py37
-    neuron_sdk_version: &NEURON_SDK_VERSION sdk1.17.1
+    neuron_sdk_version: &NEURON_SDK_VERSION sdk1.19.1
     os_version: &OS_VERSION ubuntu18.04
-    transformers_version: &TRANSFORMERS_VERSION 4.12.3
+    transformers_version: &TRANSFORMERS_VERSION 4.18.0
     tag: !join [ *VERSION, '-', 'transformers', *TRANSFORMERS_VERSION, '-', *DEVICE_TYPE, '-', *TAG_PYTHON_VERSION,"-", *NEURON_SDK_VERSION, '-', *OS_VERSION ]
     docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /, *NEURON_SDK_VERSION, /Dockerfile., *DEVICE_TYPE ]
     context:
```

Comment:
Also please add a buildspec-1-10-2-neuron.yml with the same contents in buildspec-neuron.yml

Reply:
why? isn't that something we do when we release 1.11?